### ISSUE TITLE
Unable to generate personal shareable links for 6+ months

### ISSUE THREAD

**Issue by tombinga** [pieces-app/support#443]
### Software

Desktop Application, Web Extension, VS Code, Sharing Web App

### Operating System / Platform

Windows

### Your Pieces OS Version

10.1.12

### Early Access Program

- [ ] Yes, this is related to an Early Access Program feature.

### Kindly describe the bug and include as much detail as possible on what you were doing so we can reproduce the bug.

I've been unable to generate shareable personal links and utilize the web sharing app since before June '24. I've created an issue about this twice before, and the last time was just completely disregarded and it was closed out.

Here is the screenshot from within my Pieces Desktop when trying to generate a personal link, as well as generate a GitHub Gist:

![image](https://github.com/user-attachments/assets/0966a1a9-3076-4d9b-9006-88a14673cc9f)

**Comment by tsavo-at-pieces** [comment]
@mark-at-pieces tomorrow this might be worth checking into.

**Comment by mark-at-pieces** [comment]
@tombinga completely understand your frustration, would you mind adding some time on my calendar? ([link](https://calendar.app.google/qnV9MNshBY7hGQc3A))

I can give you an update on the current status of shareable links, share our road map for the rest of the year, and I will diagnose your personal cloud and why shareable links are failing.

**Comment by tombinga** [comment]
@mark-at-pieces I've booked a slot for tomorrow 11/8. Thanks!

**Comment by mark-at-pieces** [comment]
Update: the issue is related to stale infrastructure that is powering shareable links, if any one is still having issues w/ links do NOT hesitate to schedule some time w/ me to clean up your cloud. However please stay tuned for major upgrades to come to Shareable links + additional cloud features roughly around December(in development at the moment!)

**Comment by mark-at-pieces** [comment]
this issue + fix: should help w/ the stability of this feature

https://github.com/pieces-app/os_server/issues/1306

however, there will be much more stability + infra upgrades to come here hopefully in dec b4 the holidays

**Comment by rutvik-at-pieces** [comment]
@tombinga Could you please check out the latest build of Pieces OS `v11.0.3`? and check if the issues you were facing before are resolved there?

**Comment by tombinga** [comment]
@rutvik-at-pieces I've been able to generate a shareable link (Personal Link), but when I try to generate a GitHub Gist I still get this error:

 ![Image](https://github.com/user-attachments/assets/a5b49a2e-abea-4614-a149-ddee4c9803fe)

**Comment by rutvik-at-pieces** [comment]
aghhh, that's not good. Was this gh gist issue present before you updated to the latest builds of Pieces OS and desktop app? 

I've been trying to reproduce this on my end but no luck. @tombinga Could you please file another issue(lmk if not possible, I can file it and tag you) with the issues you're facing and the following details before we can dive further into this? 

1.  Production logs, 

    if macOS: User/Library/com.pieces.os/production/Support/logs
    
    if Windows: C://User/Documents/com.pieces.os/production/Support/logs

2. Pieces OS and desktop app version.


Closing this issue as the original issue seems to have been resolved and we can continue discussing this gh gist issue separately.

**Comment by Om-Nandurkar** [comment]
I am also getting the same issue. How do I solve this error?

**Comment by mark-at-pieces** [comment]
> I am also getting the same issue. How do I solve this error?

hey @Om-Nandurkar sorry to hear you are having issues with Shareable links.

We have had issues for this feature for quite sometime even after patching this a few times

We are scoping a v2 to this and then bringing the feature out of Beta.

However in the mean time would you mind providing me with the email you use to signin to Pieces?

**Comment by Om-Nandurkar** [comment]
Do I share my email here itself?

**Comment by Om-Nandurkar** [comment]
@mark-at-pieces Here is my email  omnandurkar2004@gmail.com

**Comment by Om-Nandurkar** [comment]
Please help me with this issue. This is my email.
***@***.***

**Comment by mark-at-pieces** [comment]
hey @Om-Nandurkar I see that you have signed in but it does not look like you have connected your account to the cloud

would you mind schedule a call with me on my calendar here https://calendar.app.google/73ri52YMVyMDM8iWA
